httplug/usage.rst

```diff
@@ -0,0 +1,12 @@
+HTTPlug usage
+=============
+
+HTTPlug is relevant for three groups of users:
+
+.. toctree::
+   :maxdepth: 1
+
+   Library users <users>
+   Application developers <application-developers>
+   Library developers <library-developers>
```

Comment:
What does library developer and user mean and how is the term library compared to reusable package?